lib/coinbase/webhook.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Webhook.
+  class Webhook
+    attr_reader :webhook_id
+
+    class << self
+      # Creates a new Webhook on the specified Network and generate a default address for it.
+      # @param network_id [String]  the ID of the blockchain network.
+      # @return [Coinbase::Webhook] the new Webhook
+      def create(network_id:, notification_uri:, event_type:, event_filters:)
+        model = Coinbase.call_api do
+          webhooks_api.create_webhook(
+            create_webhook_request: {
+              network_id: Coinbase.normalize_network(network_id),
+              notification_uri: notification_uri,
+              event_type: event_type,
+              event_filters: event_filters
+            }
+          )
+        end
+
+        @webhook_id = model.id
```

Comment:
```suggestion
```

You should not set instance variables from a class method as this will create a class instance variable, which is not what you want.

Reply:
i see, then we might need to tweak the current design since webhook id is the value returned from webhook backend instead of passing from a user.

i think we probably has to do something like this
```
webhook = Coinbase::Webhook.create({ ... })

webhook.update(webhook.id, ....)
webhook.delete(webhook.id)
```

Reply:
it might not be intuitive though... welcome for any suggestions